src/components/Menu/ActionImageUpload.vue

```diff
@@ -24,6 +24,7 @@
 		:data-text-action-entry="actionEntry.key"
 		:title="actionEntry.label"
 		:aria-label="actionEntry.label"
+		role="menu"
```

Comment:
Shall we upstream this to https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/Actions/Actions.vue#L136 ?

Reply:
Makes sense to me.